CONTRIBUTING.md

```diff
@@ -6,9 +6,12 @@ Work done surrounding this project is governed by the
 
 ## Dependencies
 The `requirements.txt` file is frozen to the currently tested
-versions of `requirements.base`. To update, create a venv,
-run `python3 -m pip install -r requirements.base`, run all tests,
-and then update `requirements.txt` with the output from `pip freeze`.
+versions of `requirements.in`. To update, create a venv,
+install [`pip-tools`](https://pypi.org/project/pip-tools)
+run `python3 -m pip-sync`, run all tests, and then update 
```

Comment:
`python3 -m pip-sync` doesn't work for me. I needed to do just `pip-sync`.

Maybe easier if these instructions are in bullet points?